### PR TITLE
HYDRA-805 - add test for look through camera&lights

### PR DESCRIPTION
**Pull request by lilike-adsk**
add test for look through camera&lights to cover
1.look through the camera
2.look through the spot light
3.look through the area light with clip planes

### CODE REVIEW

**Comment by lilike-adsk** [comment]
FYI @lanierd-adsk  @roopavr-adsk @ppt-adsk @vlasovi

File: test/lib/mayaUsd/render/mayaToHydra/testLookThrough.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    def imageDiffFailPercent(self):
+        # HYDRA-837 : Wireframes seem to have a slightly different color on macOS. We'll increase the thresholds
+        # for that platform specifically for now, so we can still catch issues on other platforms.
+        if platform.system() == "Darwin":
+            return 3
+        return 0.2
+
+    #Test look through camera and maya lights (spot & area).
+    def test_LookThrough(self):
+        cmds.file(new=True, force=True)
```
Comment (debloip-adsk) [inline test/lib/mayaUsd/render/mayaToHydra/testLookThrough.py]: This is no longer necessary, as now a new scene will be opened before each individual test.

File: test/testSamples/testStandardSurface/testStandardSurface.ma [hunk 2]
```diff
@@ -112,9 +111,6 @@ createNode file -n "file1";
 	setAttr ".cs" -type "string" "sRGB";
 createNode place2dTexture -n "place2dTexture1";
 	rename -uid "6616C90D-4B36-08C7-8666-33BB13D15663";
-createNode mayaUsdLayerManager -n "mayaUsdLayerManager1";
-	rename -uid "61CE7C31-4E04-09B7-BFEA-F291A58D7E4C";
-	setAttr ".sst" -type "string" "";
```
Comment (debloip-adsk) [inline test/testSamples/testStandardSurface/testStandardSurface.ma:117]: Are these changes required? Not sure why they're here, doesn't look like they're used

**Comment by debloip-adsk** [comment]
Thanks!  Two minor things, otherwise looks good, just waiting to see if the changes in testStandardSurface.ma are okay to have in before approving

File: test/testSamples/testStandardSurface/testStandardSurface.ma (hunk 2 above)
Comment (lilike-adsk) [reply test/testSamples/testStandardSurface/testStandardSurface.ma:117]: This's just for clearing out mayausd stuff, this might cause loading of mayausd, which is not necessary for this case.

File: test/lib/mayaUsd/render/mayaToHydra/testLookThrough.py (hunk 1 above)
Comment (lilike-adsk) [reply test/lib/mayaUsd/render/mayaToHydra/testLookThrough.py]: good, I'll remove it

**Review by debloip-adsk — APPROVED** [review]
